LICENSE.txt

```diff
@@ -0,0 +1,614 @@
+
```

Comment:
just stick with a base ALv2 in here for now, presuming we are not yet including any outside stuff in the source itself. we can work out what is needed in LICENSE for the binary convenience artifacts when we have a better idea what that looks like.

Reply:
Not sure if there's an action item requested here. This is a base ALv2...

Reply:
There's a bunch of other stuff starting around line 205

Reply:
Oh you're right I had missed that. Copied the LICENSE from HBase - which I notice is also executable :) Fixing...